src/app/components/game-components/abstract-game-component/AbstractGameComponent.spec.ts

```diff
@@ -114,4 +118,18 @@ describe('AbstractGameComponent', () => {
         }
         flush();
     }));
+    it('Component should have a encoder and a static tutorial', fakeAsync(async() =>{
```

Comment:
an encoder

Reply:
By the way, second modification done on the title, the tutorial is not static.